src/main/java/lotto/domain/LottoMachine.java

```diff
@@ -17,16 +17,10 @@ public class LottoMachine {
         }
     }
 
-    public static BuyLotto createLotto(LottoPrice lottoPrice) {
-        List<Lotto> lottoNumbers = new ArrayList<>();
-
+    public static void createLotto(BuyLotto buyLotto, LottoPrice lottoPrice) {
```

Comment:
```suggestion
    public static BuyLotto createLotto(LottoPrice lottoPrice) {
```
이 메서드는 BuyLotto를 생성해 반환하는 방식으로 구현하면 어떨까?
메서드 이름은 무엇인가 값을 반환하는 것으로 보여짐

위와 같이 구현한 후 두개의 ArrayList를 합칠 때 addAll(ArrayList values)와 같은 메서드로 추가하는데 위와 같이 생성한 후에 수동, 자동의 List를 합치는 또 다른 메서드를 추가하는 것은 어떨까?